docs/development_suite/api-console/api-design/microservice-containers.md

```diff
@@ -0,0 +1,89 @@
+
+For these reasons the sidecar-pattern comes very handy. You can develop a lightweight service that specializes in logging requests to the main application and store these logs in a centralized logging system.  
+You can then deploy this logging service along with the main application on the same host. This ensures that the sidecar and main service containers are always running on the same machine.
+
+In this section we will go deeper in how you can manage this kind of containers for your microservices.
+
+
+## Microservice detail page
+
+The containers table in the microservice detail page gives an immediate overview of all the containers that your microservice is made of, indicating which is the **main container** and which is the **sidecar** one.
+
+The table shows a summary of `requests` and `limits` set for both container CPU and memory resources, as well as which ConfigMaps and Secrets are used by which container. From here it is also possible to add and delete sidecars from the current microservice.
+
+![Microservice Containers](img/microservice-containers/containers-table.png)
+
+### Exposing a specific container
+
+:::info
+Exposing a container means making it reachable from other workloads inside the same Project environment.  
+By default all the containers that specify a container port are exposed by the Console if no other changes are made.
+:::
+
+From the containers table you can decide whether to expose all your service containers or toggle only the ones you want to expose. In the example below, only the **main container** is configured to be exposed.
+
+In order to expose a container, this must define at least one container port. In case no container ports are defined, the exposure switch will be disabled. Configure ports from the specific section in the [sidecar detail](#sidecar-detail)
+
+![Exposing containers](img/microservice-containers/exposing-containers.png)
+
+### Sidecar detail
+
+Clicking on the sidecar name in the table, you enter the sidecar configuration page.  From this page you can inspect and update the configuration as well as adding configMaps, secrets, update the docker image and all the other settings, the same way as you can edit a microservice configuration.
+
+
+:::info
+The main container and other [special sidecar containers](#special-sidecars) cannot be removed nor modified from this section. These containers are directly managed from other sections of the Console. Read below for more.
+:::
+
+In this example you can see the same sections that are present in the microservice detail too. To correctly configure these sections check out the relative documentation:
+
+- [General](/development_suite/api-console/api-design/services.md#manage-microservices)
+- [Environment Variables](/development_suite/api-console/api-design/services.md#environment-variable-configuration)
+- [ConfigMaps & Secrets](/development_suite/api-console/api-design/services.md#custom-configuration)
```

Comment:
I would add a brief description of what these sections does. Might sound redundant but it would help give context to the page.